py/selenium/webdriver/common/print_page_options.py

```diff
@@ -0,0 +1,154 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class PrintOptions():
+
+    def __init__(self):
+        self.orientation_values = ['portrait', 'landscape']
+        self._print_options = {}
+        self._page = {}
+        self._margin = {}
+
+    @property
+    def to_dict(self):
+        """
+        :Returns: A hash of print options configured
+        """
+        return self._print_options
+
+    def set_orientation(self, value):
```

Comment:
We should have these are getter and setter properties rather than using `set_*` methods.

See an `Options` class as an example